pdm/src/main/java/me/bristermitten/pdm/util/ClassLoaderReflection.java

```diff
@@ -10,38 +10,39 @@
 public class ClassLoaderReflection
 {
 
-    private static final Method addUrlMethod;
+    private static final Method ADD_URL_METHOD;
 
     static
     {
-        Method addURL;
+        final Method addURL;
+
         try
         {
             addURL = URLClassLoader.class.getDeclaredMethod("addURL", URL.class);
             addURL.setAccessible(true);
         }
-        catch (NoSuchMethodException e)
+        catch (NoSuchMethodException exception)
         {
-            addURL = null;
-            e.printStackTrace();
+            throw new AssertionError(exception);
         }
-        addUrlMethod = addURL;
+
+        ADD_URL_METHOD = addURL;
     }
 
     private ClassLoaderReflection()
     {
-
+        throw new AssertionError("This class cannot be instantiated.");
```

Comment:
Meh, this feels unnecessary to me, but alright.

Reply:
it is unnecessary, but so it having 4 lines for an empty constructor.

Reply:
That's just Java though

Reply:
no, that's your code style

Reply:
In K&R it would still be 3 lines. 

As far as I'm concerned, having an exception in a constructor that's impossible to call without reflection anyway is just unnecessary boilerplate. It doesn't achieve anything, and the lack of instantiation should be clear from the fact it's a Utils class with a private constructor.

Reply:
I’m not sure if k&r allows this, but I personally do it all on one line. CoolClass() {}